src/stories/3-fields/date-range-field/date-range-field.stories.tsx

```diff
@@ -169,6 +170,20 @@ PastDateOnly.args = {
 	validation: [{ required: true }, { past: true, errorMessage: "Only past dates allowed" }],
 };
 
+export const NotFutureDate = DefaultStoryTemplate<TDateRangeFieldSchema>("date-not-future").bind({});
+NotFutureDate.args = {
+	uiType: "date-range-field",
+	label: "Date",
+	validation: [{ required: true }, { notFuture: true, errorMessage: "No future dates allowed" }],
+};
+
+export const NotPastDate = DefaultStoryTemplate<TDateRangeFieldSchema>("date-not-past").bind({});
+NotFutureDate.args = {
```

Comment:
minor issue, the story is not showing up properly
```suggestion
NotPastDate.args = {
```